src/commands.ts

```diff
@@ -470,8 +473,17 @@ export const insertInlineFootnote =
       // inside footnote inside element inside section
       selectionPos = backmatter.pos + section.nodeSize - 3
     } else {
-      const element = elements[0]
-      const pos = element.pos + element.node.nodeSize - 1
+      // Look for footnote element inside the footnotes section to exclude tables footnote elements
+      const footnoteElement = findChildrenByType(
+        footnotesSection.node,
+        schema.nodes.footnotes_element
+      )
+      // TODO: Revisit this position calculation as it doesn't sound right to always push the note to the end.
+      const pos =
+        footnotesSection.pos +
+        footnoteElement[0].pos +
```

Comment:
hm, isn't footnoteElement[0].pos is supposed to be a global position? So it shouldn't require "footnotesSection.pos" addition. Am I missing something?

Reply:
Maybe you missed the comment in line 476.

Reply:
Ah, looked up the utils source code and I see now that it's a position relative to the node in which the search was conducted.